src/sections/ui/badge/Badge.module.scss

```diff
@@ -0,0 +1,27 @@
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+@import 'src/sections/ui/assets/styles/design-tokens/colors.module';
+
+.primary {
+    color: color-contrast($dv-primary-color) !important;
+}
```

Comment:
I thought we were trying to avoid using `!important`. Should we investigate CSS layers?

Reply:
I added the CSS layers and got rid of the `!important`. Thanks!